gradle-remote

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Please note that this script is quite basic and not recommended for use with public build servers.
```

Comment:
If we're adding this file top-level instead of hidden in the build scripts folder, I think we should `echo` this message and prompt before continuing, optionally with a `-y` argument or env var to skip the prompt.

Reply:
I think adding a prompt defeats the purpose of a drop-in replacement for Gradle calls.

Reply:
I'm thinking more of the first time use case. If you use an environment variable to subvert the prompt, then you can simply set `GRADLE_REMOTE_ACCEPT_PITFALLS=y` or something like that and you can use it as a drop-in.